presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/InternalResourceGroup.java

```diff
@@ -82,6 +82,8 @@
 
     private final InternalResourceGroup root;
     private final Optional<InternalResourceGroup> parent;
+    private Optional<InternalResourceGroup> next;
```

Comment:
nits: Move this to be together with other not final member variables.

Reply:
done